compiler/noirc_evaluator/src/ssa/opt/remove_if_else.rs

```diff
@@ -283,6 +298,11 @@ fn slice_capacity_change(
         }
 
         Intrinsic::SlicePopBack | Intrinsic::SliceRemove => {
+            // fn pop_back(self) -> (Self, T)
+            // fn remove(self, index: u32) -> (Self, T)
+            //
+            // These functions return the slice as the first result `(len, slice)`,
```

Comment:
```suggestion
            // These functions return the slice as the result `(len, slice, item)`,
```
It took me a long time to understand how `(Self, T)` becomes `(len, slice)`, it looked like they just swapped places.